brainstormforce/astra
# AST-1318 - Added compatibility settings for fixed switch addon

### Description
- Added new Fixed Switch addon from Astra Pro
- Here is PR for making compatible pro addon
- Also code for footer builder component

### Types of changes
- New feature non-breaking change 

### How has this been tested?
- Check with addon's fixed switch
- Check with footer builder dark mode element

### Checklist:
